aws-lambda-function/main.tf

```diff
@@ -62,7 +62,8 @@ EOF
 }
 
 resource aws_cloudwatch_log_group log {
-  name = "/aws/lambda/${local.name}"
+  name              = "/aws/lambda/${local.name}"
```

Comment:
unrelated but: should the aws_lambda_function `depends_on` this `aws_cloudwatch_log_group` ?

Reply:
yeah probably